Frontend/app/class/detail/detail.html

```diff
@@ -1,6 +1,8 @@
 <div class="row">
   <div class="col-xs-12">
-    <h2>Class Detail Page</h2>
+    <h3>[Class Name] <small>[Class Department]</small></h3>
+    <h4><small><strong>[Class Section]</strong></small></h4>
+    <p>[Class Description]</p>
```

Comment:
Why is this and the addition to class.service.ts on this pull request? What does it do?